.gitattributes

```diff
@@ -1 +1,4 @@
+# Ensure line endings are the same on all systems to prevent "outdated" error while uploading recipe
+* text=auto eol=lf
```

Comment:
This will only affect new files, wouldn't it?

Or are the line endings of existing files fine anyway?

Reply:
I think the files are fine and it only affects the line endings on clone / checkout on Windows.